webapp/pages/profile/_id/_slug.vue

```diff
@@ -199,18 +199,16 @@
           </template>
           <template v-else>
             <p style="text-align: center; opacity: .5;">
-              niemand folgt {{ user.name }}
+              niemand folgt {{ userName }}
             </p>
           </template>
         </ds-card>
-        <ds-space 
+        <ds-space
           v-if="user.socialMedia && user.socialMedia.length"
           margin="large"
         >
           <ds-card style="position: relative; height: auto;">
-            <ds-space
-              margin="x-small"
-            >
+            <ds-space margin="x-small">
               <ds-text
                 tag="h5"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-17T14:06:57Z" title="Wednesday, April 17th 2019, 4:06:57 pm +02:00">Apr 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -430,6 +430,14 @@ export default {
         },
         fetchPolicy: 'cache-and-network'
       })
+    },
+    userName(maxLength) {
+      // Return Anonymous if no Username is given
+      if (!this.user.name) {
+        return this.$t('profile.userAnonym')
+      }
+      // Return full Username or truncated Username
+      return maxLength ? this.user.name.substring(0, maxLength) : this.user.name
```


Actually I like `truncate` more. Does it add ellipsis ... when you have a long user name?

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-19T09:51:34Z" title="Friday, April 19th 2019, 11:51:34 am +02:00">Apr 19, 2019</time>_

---


![image](https://user-images.githubusercontent.com/1238238/56419111-755c1c80-6299-11e9-95cf-f8dd5a8116cd.png)

using truncate now